Bugsnag/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_Signal.c

```diff
@@ -212,6 +212,11 @@ bool bsg_kscrashsentry_installSignalHandler(
             }
             goto failed;
         }
+        if (fatalSignals[i] == SIGPIPE &&
```

Comment:
I wonder if maybe we should check for SIG_IGN on all signals except KILL and STOP (which can't be ignored).

Reply:
Yes that might be worth a follow-up 👍